.devcontainer/Dockerfile

```diff
@@ -1,3 +1,14 @@
 FROM julia:latest
 
-RUN apt-get update && apt-get install -y build-essential libatomic1 python gfortran perl wget m4 cmake pkg-config git
+RUN apt-get update \
+    install -y \

```

Comment:
This should be on the same line as `apt-get`?

Reply:
And also need `apt-get` before `install`. The only thing "wrong" with the file from the start is perhaps that they recommend sorting the packages (which isn't even done). The last line should not end with `\` either. As a general rule it is good to try out changes you make before submitting pull request and having people spend time reviewing.